src/main/java/com/google/sps/servlets/IDVerifyer.java

```diff
@@ -0,0 +1,75 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.Collections;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdTokenVerifier;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
+import com.google.sps.util.CredentialManager;
+
+import com.google.sps.util.OAuth2Credentials;
+
+@WebServlet("/Verify")
+public class IDVerifyer extends HttpServlet {
+
+    /**
+     *
+     */
```

Comment:
Missing javadoc?

Reply:
My apologies, this was an auto-generated javadoc for the serialUID, this has been cleared up